api/serializers.py

```diff
@@ -376,3 +375,9 @@ def validate(self, fields):
                 "Similar asset specification already exist"
             )
         return fields
+
+
+class BuildingSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Building
+        fields = ("block_name", )
```

Comment:
I think we should include the `id` as well

Reply:
A custom `id` field?
Remember Django adds an `id` field as the primary_key by [default](https://docs.djangoproject.com/en/2.0/topics/db/models/#automatic-primary-key-fields).

Reply:
@egichuri Is it inadvisable to depend on Django's default `id` field??

Reply:
it's ok to use the default id.. but let's expose it on fields available to the API

Reply:
Got it 👍